Deployment/src/Modules/FulfilmentKeyVault/main.tf

```diff
@@ -0,0 +1,67 @@
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_key_vault" "small_exchange_set_kv" {
```

Comment:
Do we plan to have separate scripts for small, medium and large ES KVs? Their configurations are going to be very identical. Current script is very much reusable, just need to add a parameter for take KV name. Please can we try to make script common.
This is applicable for storage account too.

Reply:
I won't be adding separate scripts but add medium and large storage and keyvault on same fulfilment module scripts but as different resource. I kept like this so that resources related to fulfilment are created within same module. Rather calling individual from main module.